packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -36,6 +36,7 @@ import { DatepickerCalendarHeader } from './DatepickerCalendarHeader';
 import { DatepickerCalendarHeaderAction } from './DatepickerCalendarHeaderAction';
 import { DatepickerCalendarHeaderCell } from './DatepickerCalendarHeaderCell';
 import { validateDate } from './validateDate';
+import { format } from '@uifabric/utilities';
```

Comment:
Adding the dependency on `@uifabric/utilities` is likely to cause build problems at the moment, so you may need to either move the `format` function (from `packages/utilities/src/string.ts`) to an existing shared location if a suitable one exists, or make a local copy for now.

Reply:
Can you please expand why the dependency would cause problems?

The @uifabric/utilities package is already in the dependency chain within the fluentui/react-bindings. So the overall's react-northstar's package flattened dependencies do not change.

Reply:
Oh okay I hadn't realized that. In that case it should be fine. The reason I was cautious in general is we've run into some difficulty importing v7 packages from v0 in that past, but that may be resolved now.

Reply:
I understand now.

Thanks a lot for explaining the background. :)